.github/actions/startserver-action/startCIServer.sh

```diff
@@ -56,6 +60,11 @@ function launchOpenfire {
 	# Replace the default XMPP domain name ('example.org') that's in the demoboot config with the configured domain name.
 	sed -i -e 's/example.org/'"${HOST}"'/g' ${BASEDIR}/conf/openfire.xml
 
+	# Replace the default log level ('info') that's in the log4j config with the configured level.
+	if [[ -n "${LOGLEVEL}" ]]; then
+	    sed -i -e 's/info/'"${LOGLEVEL}"'/g' ${BASEDIR}/lib/log4j2.xml
```

Comment:
This feels rather brittle to me and the `/g` should definitely not be included.

Reply:
How about now?